src/Default/default.ts

```diff
@@ -108,8 +111,10 @@ export class DefaultMessageBus implements MessageBus {
     publish<T>(topic: string, message: T, options?: MessageBusOptions): Promise<void> {
         // Get list of windows from global (set by opener on creation) or fallback to opener global in case
         // there is a race condition of getting here before the opener set the global for us
-        const windows: Window[] = this.container.globalWindow[DefaultContainer.windowsPropertyKey]
```

Comment:
Added some conditional checks for safety that were more for unit test mocking then production use but good to have regardless.